_shared.py

```diff
@@ -188,6 +182,12 @@ def parse_args(*, prog_desc: str, out_to_json: bool = False,
     args = parser.parse_args()
     argv = sys.argv[1:]
 
+    if args.edgedb_port is None:
+        # read the port from the `_edgedb/bench_cfg.json`
```

Comment:
Why is this hackery necessary?  All our bindings should be able to discover the instance address/port from the project link.

Reply:
Yes. See above comment on instance names.